src/play/view/PygameGuiView.py

```diff
@@ -39,18 +39,23 @@ def open(self, game_controller):
         self.running = True
         self.screen = pygame.display.set_mode(window_size)
         pygame.display.set_caption('Go')
-        self.buttons.append(Button(210, 530, 80, 40, 'Pass', self.screen, self.send_pass_move))
-        self.labels.append(Label(100, 30, 300, 40, self.get_turn_label_text, self.screen))
+        self.buttons.append(Button(
+            210, 530, 80, 40, 'Pass', self.screen, self.send_pass_move))
+        self.labels.append(Label(
+            100, 30, 300, 40, self.get_turn_label_text, self.screen))
         self.render()
 
         while self.running:
             event = pygame.event.poll()
             if event.type == pygame.MOUSEBUTTONUP:
                 x, y = event.pos
-                col = int(round((x - board_top_left_coord[0]) / self.cell_size))
-                row = int(round((y - board_top_left_coord[1]) / self.cell_size))
+                col = int(round((x - board_top_left_coord[0]) /
+                                self.cell_size))
+                row = int(round((y - board_top_left_coord[1]) /
+                                self.cell_size))
```

Comment:
Would it be too long otherwise? In PyCharm I try to break lines only when I reach the gray line, on the right side of the screenshot:
![screen shot 2017-11-06 at 21 08 35](https://user-images.githubusercontent.com/5141792/32461605-b41501ca-c336-11e7-829c-f0da04a929d9.png)
I thought PyCharm does follow PEP8 🤔

Reply:
PEP8 convention are 79 characters max per line. I think PyCharm defaults to something else then? But I guess you can change that somewhere.